src/dropdown/dropdown.component.ts

```diff
@@ -35,24 +35,35 @@ import { I18n } from "./../i18n/i18n.module";
 			'bx--list-box--inline': inline,
 			'bx--skeleton': skeleton
 		}">
-		<button
-			type="button"
+		<div
 			#dropdownButton
 			class="bx--list-box__field"
-			[ngClass]="{'a': !menuIsClosed}"
+			[ngClass]="{'a': !menuIsClosed, 'no-selections': type === 'single' || ( !view.getSelected() || !view.getSelected().length )}"
 			[attr.aria-expanded]="!menuIsClosed"
 			[attr.aria-disabled]="disabled"
-			(click)="toggleMenu()"
 			(blur)="onBlur()"
-			[disabled]="disabled">
-			<span class="bx--list-box__label">{{getDisplayValue() | async}}</span>
-			<div *ngIf="!skeleton" class="bx--list-box__menu-icon" [ngClass]="{'bx--list-box__menu-icon--open': !menuIsClosed }">
-				<svg fill-rule="evenodd" height="5" role="img" viewBox="0 0 10 5" width="10" [attr.aria-label]="menuButtonLabel">
-					<title>{{menuButtonLabel}}</title>
-					<path d="M0 0l5 4.998L10 0z"></path>
+			>
+			<span (click)="clearSelected()" *ngIf="type === 'multi' && view.getSelected() && view.getSelected().length"
+				class="bx--tag bx--tag--ibm">
```

Comment:
This should be `bx--list-box__selection bx--list-box__selection--multi` ... dropdown already has specific styles for this in the carbon-components css :+1: be sure to check out the rendered output of https://v9.carbondesignsystem.com/components/dropdown/code (specifically the multiselect) to see the structure